scripts/tags/exturl.js

```diff
@@ -48,19 +43,18 @@ function extURL(args, content) {
   // If any arguments exists, collect the last text as the link title,
   // if not, set title as url.
   if (args.length) {
-    var shift = args[0];
     title = args.join(' ');
   } else {
     title = item;
   }
 
   var attrs = {
-    class: exturl,
+    class     : exturl,
     'data-url': url,
-    title: title
+    title     : title
   };
 
-  return htmlTag('span', attrs, text.join(' '));
+  return htmlTag('span', attrs, text.join(' ') + '<i class="fa fa-external-link"></i>');
```

Comment:
IMO, this should be an option to let user decide.

Reply:
Emm.. can u add this option?

Reply:
Maybe after this PR is merged into master.

Reply:
Ok.